### PR TITLE
Add system test to load dashboards

### PR DESCRIPTION
**Pull request by ruflin**
* Move test_version test to base tests as it should not be in base.py

### CODE REVIEW

File: dev-tools/import_dashboards.sh [hunk 1]
```diff
@@ -171,10 +171,10 @@ fi
 if [ -d "${DIR}/index-pattern" ]; then
     for file in ${DIR}/index-pattern/*.json
     do
-        NAME=`awk '$1 == "\"title\":" {gsub(/[",]/, "", $2); print $2}' ${file}`
```
Comment (monicasarbu) [inline dev-tools/import_dashboards.sh]: I wouldn't remove this as it gets the name of the index pattern from the `title` field available in the json file, that can be different than the name of the file.  Think of `packetbeat-*` that is available under packetbeat.json

File: libbeat/tests/system/test_dashboard.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
+from base import BaseTest
+
+import os
+import subprocess
+
+
+class Test(BaseTest):
+    def test_load_dashboard(self):
```
Comment (monicasarbu) [inline libbeat/tests/system/test_dashboard.py]: great idea to add a test for loading the dashboards.

File: dev-tools/import_dashboards.sh (hunk 1 above)
Comment (ruflin) [inline dev-tools/import_dashboards.sh]: @monicasarbu I changed it as the code did not work anymore. I changed it now to `${NAME}-*`. This is only a temporary fix and this script should be removed when https://github.com/elastic/beats/pull/2155 is merged.

**Comment by ruflin** [comment]
@monicasarbu I just realised that even though I made the tests compatible with windows, it does not run on windows as our libbeat tests are not run on windows. As these tests need the full environment, it is good to run them in libbeat. As the code with https://github.com/elastic/beats/pull/2155 will be identical for windows and others, probably it will be sufficient to run them only on linux systems.

File: dev-tools/import_dashboards.sh (hunk 1 above)
Comment (monicasarbu) [inline dev-tools/import_dashboards.sh]: I have found the problem. #2217 is fixing this issue.

**Comment by monicasarbu** [comment]
The failing test should be fixed in https://github.com/elastic/beats/pull/2220.

**Comment by ruflin** [comment]
@monicasarbu I merged #2220 and rebased on top of master.

**Pull request merged**